src/app/vendors/Vendors.tsx

```diff
@@ -119,6 +122,12 @@ class Vendors extends React.Component<Props, State> {
       this.setState({ error: undefined });
     }
 
+    if ($featureFlags.vendorEngrams) {
+      dimVendorEngramsService
+        .getAllVendorDrops()
```

Comment:
It's a bit awkward that we're driving this from two intervals (one in the header and one here).

Reply:
`Vendors.tsx` doesn't set up an interval (`Header.tsx` does), but I guess reloading the page with vendors open could lead to a pathological state where both components trigger a fetch.

Listening to Redux updates would fix that.